docs/doc/14-sql-commands/00-ddl/91-catalog/hive.md

```diff
@@ -7,6 +7,27 @@ import FunctionDescription from '@site/src/components/FunctionDescription';
 
 Databend supports the integration of an [Apache Hive](https://hive.apache.org/) catalog, enhancing its compatibility and versatility for data management and analytics. This extends Databend's capabilities by seamlessly incorporating the powerful metadata and storage management capabilities of Apache Hive into the platform.
 
+## Datatype Mapping to Databend
+
+This table maps data types between Apache Hive and Databend. Please note that Databend does not currently support Hive data types that are not listed in the table.
+
+| Apache Hive         | Databend             |
+| ------------------- | -------------------- |
+| boolean             | boolean              |
```

Comment:
It's better upper-case the data types, and add a link which we have introduced in sql-reference/data-types/ chapter for Databend.